src/components/carousel/CarouselList.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <section id="carousel">
+    <div class="container">
+      <div class="carousel-wrap" ref="carouselWrap">
+        <div class="carousel-list" ref="carouselList">
+          <carousel-item
+            v-for="(item, index) in carouselList"
+            :key="item.id"
+            :item="item"
+            :index="index"
+            ref="carouselItem"
+          ></carousel-item>
+        </div>
+      </div>
+      <div class="carousel__navigation">
+        <div class="carousel__pagination">
+          <span
+            v-for="(item, index) in carouselList"
+            :key="item.id"
+            :class="{
+              'pagination-bullet pagination-bullet--active': index === 0,
```

Comment:
Change implementation to declarative to avoid manual setting of attributes.
Consider passing `active` prop to the carousel-item as well.